app/src/main/java/mozilla/lockbox/action/TelemetryAction.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package mozilla.lockbox.action
+
+import mozilla.lockbox.flux.Action
+import org.mozilla.telemetry.event.TelemetryEvent
+
+open class TelemetryAction(
+    val eventMethod: TelemetryEventMethod,
+    val eventObject: TelemetryEventObject,
+    val value: String?,
+    val extras: Map<String, Any>?
+) : Action {
+    open fun createEvent(category: String = "action"): TelemetryEvent {
+        val evt = TelemetryEvent.create(
```

Comment:
ohh, default implementations are not a thing in Kotlin...

Reply:
Default implementations on an interface are not a Java thing, so they're not a Kotlin thing.

I take it this is enough reason to leave it as `open class TelemetryEvent` instead of `interface TelemetryEvent`?